test/gtest/ucs/test_memtype_cache.cc

```diff
@@ -233,6 +317,76 @@ UCS_TEST_P(test_memtype_cache, basic) {
     test_ptr_not_found(ptr, size);
 }
 
+UCS_TEST_P(test_memtype_cache, update_non_contig_regions_and_remove_subintervals) {
+    std::vector<test_memtype_cache::region_info> insert_regions;
+    std::vector<test_memtype_cache::region_info> remove_regions;
+    size_t start, end;
+
+    const size_t region_size           = std::pow(2, 28);
+    const size_t interval_start_offset = std::pow(2, 27);
+
+    // insert [0x7f6ef0000000 .. 0x7f6f00000000]
+    start = 0x7f6ef0000000;
+    end   = start + region_size;
+    test_memtype_cache::region_info region_info1 =
+        make_region_info(start, end, GetParam());
+    insert_regions.push_back(region_info1);
+
+    // insert [0x7f6f2c021000 .. 0x7f6f3c021000]
+    start = 0x7f6f2c021000;
+    end   = start + region_size;
+    test_memtype_cache::region_info region_info2 =
+        make_region_info(start, end, UCS_MEMORY_TYPE_LAST);
+    insert_regions.push_back(region_info2);
+
+    // insert [0x7f6f42000000 .. 0x7f6f52000000]
+    start = 0x7f6f42000000;
+    end   = start + region_size;
+    test_memtype_cache::region_info region_info3 =
+        make_region_info(start, end, UCS_MEMORY_TYPE_LAST);
+    insert_regions.push_back(region_info3);
+
+    test_region_insert_and_remove_subintervals(insert_regions,
+                                               interval_start_offset,
+                                               0, remove_regions);
+}
+
+UCS_TEST_P(test_memtype_cache, update_adjacent_regions_and_remove_subintervals) {
+    std::vector<test_memtype_cache::region_info> insert_regions;
+    std::vector<test_memtype_cache::region_info> remove_regions;
+    size_t start, end;
+
+    const size_t region_size           = std::pow(2, 28);
```

Comment:
why not UCS_BIT(28)

Reply:
I forgot about this.
Done for all places and removed `#include <cmath>`